README.md

```diff
@@ -219,80 +219,58 @@ The corresponding function in on Tarantool Cartridge router may look like (uses
-
-```java
-@Data
-@Tuple
-public class SampleUser {
-    private String name;
-    private String lastName;
-}
-```
+##### Specify output
+For such methods, you can specify the stored function response format so that it will be parsed correctly.
+The response format is also specified in the `@Query` annotation in the output parameter.
 
 ```java
 public interface SampleUserRepository extends TarantoolRepository<SampleUser, String> {
-    @Query(function = "returning_sample_user_object")
-    SampleUser returningSampleUserObject(String name);
+    @Query(function = "get_users_with_age_gt", output = TarantoolSerializationType.TUPLE)
+    List<SampleUser> usersWithAgeGreaterThen(Integer age);
 
-    @Query(function = "get_predefined_users")
-    List<SampleUser> getPredefinedUsers();
+    @Query(function = "get_predefined_user", output = TarantoolSerializationType.AUTO)
+    SampleUser predefinedUser();
 }
 ```
 
-```java
-@Tuple
-public class Book {
-  @Id
-  private Integer id;
-  private String name;
-}
+```lua
+function get_users_with_gt_age(age)
+    return crud.select("sample_user", { { ">", "age", age } })
+end
 
-// Here, the space name is "another_space" for all @Query methods
-@Tuple("another_space") 
-public interface BookRepository extends TarantoolRepository<Book, Integer> {
-    
-    // The space name here overrides "another_space" to "test_space" for the current method
-    @Tuple("test_space") 
-    @Query(function = "do_something")
-    Optional<Book> doSomething(Book book);
-}
+function get_predefined_user()
+    return { name = "John", age = 46 }
+end
 ```
+The first function returns users who are older than the specified age.
+In the Repository, we indicate that we expect `TUPLE`, because `crud.select` returns the result in compressed format, flatten tuples.
+The keys for mapping the result into a Java object are obtained by the connector by the class name in returnType or by the specified `@Tuple` above it.
 
-Here the space name is not specified neither in a `@Tuple` annotation on the repository interface, nor on the entity class, so it will be "book_entity" according to the entity class name, turned into "lower snake case".
-But for the "doSomething" method the space name will be overridden to "test_space".
-```java
-@Tuple
-public class BookEntity {
-  @Id
-  private Integer id;
-  private String name;
-}
+The second function returns a ready-made result with keys.
+So `TUPLE` will not work here, because it's not flatten structure, and we specify `AUTO`.
+Mapping happens by keys of table.
 
-public interface BookRepository extends TarantoolRepository<BookEntity, Integer> {
-    
-    // The space name here overrides to "test_space" for the current method
-    @Tuple("test_space") 
-    @Query(function = "do_something")
-    void doSomething(Book book);
-}
+Of course, the word `AUTO` means that we can accept any result, and therefore it can be specified in the first request as well.
```

Comment:
Do we state somewhere that specifying AUTO explicitly is not needed and it is a default value?
TUPLE cannot be the default since this breaks the UX (AUTO is default in many other similar cases).

Reply:
We can think about this again, whether to make TUPLE or AUTO by default.
From the arguments that we came up with when designing the API, and which I give just below in the readme, these are:
1) TUPLE should be faster because we pass data without keys, TUPLE will teach the user to use the high perf option
2) SimpleTarantoolRepository - predefined functions work with TUPLE by default (differently don't work) and it becomes consistent the whole Repository API works with TUPLE by default

Reply:
My arguments are the following:
 - Tuple output is "native" and therefore "default" for Tarantool. It is the way how box API works. So any option that specifies the "tuple" output should be the default.
 - The name "AUTO" makes the user assume that it is a default option. So in my POV, "AUTO" should correspond to the "tuple" output.
 - I suggest adding a new output type "LUA" for the other cases since it basically is an output of a Lua function.
 - Any other types are possible to be added further since I'm not completely sure that we can detect all types automatically.

Reply:
I like the `LUA` option, but `AUTO` for tuples would be confusing. I think a good option is `TUPLE` **by default** and `LUA` to specify any other output.